generate-favicons/run.sh

```
@@ -0,0 +1,267 @@
+    exit 1
+fi
+
+print_status "Input file: $INPUT_FILE"
+print_status "Output directory: $OUTPUT_DIR"
+
+# Create output directory if it doesn't exist
+mkdir -p "$OUTPUT_DIR"
+
+# Temporary working directory
+TEMP_DIR=$(mktemp -d)
+trap "rm -rf $TEMP_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)